source_code/src/UTILS/delays.c

```diff
@@ -23,13 +23,19 @@
 *    Author:   Mathieu Stephan
 */ 
 #include "timer_manager.h"
+#include "mini_inputs.h"
 
 /*! \fn     userViewDelay(void)
 *   \brief  2secs delay for the user to view information
 */
 void userViewDelay(void)
 {
     timerBasedDelayMs(2000);
+
+#ifdef MINI_VERSION
+    // Discard user wheel input
+    miniWheelClearDetections();
```

Comment:
That'd require more flash usage for no reason.

```
[arch@arch src]$ grep userViewDelay . -r
./GUI/standard_gui_smartcard_functions.c:                    userViewDelay();
./GUI/standard_gui_basic_functions.c:            userViewDelay();
./GUI/standard_gui_screen_functions.c:                        userViewDelay();
./GUI/standard_gui_screen_functions.c:                        userViewDelay();
./GUI/standard_gui_screen_functions.c:                userViewDelay();
./GUI/standard_gui_screen_functions.c:                userViewDelay();
./GUI/standard_gui_screen_functions.c:    userViewDelay();
./GUI/mini_gui_basic_functions.c:            userViewDelay();
./GUI/mini_gui_screen_functions.c:            userViewDelay();
./GUI/mini_gui_screen_functions.c:                    userViewDelay();
./GUI/mini_gui_screen_functions.c:                    userViewDelay();
./GUI/mini_gui_screen_functions.c:    userViewDelay();
./LOGIC/logic_smartcard.c:    userViewDelay();
./USB/usb_cmd_parser.c:                        userViewDelay();
./USB/usb_cmd_parser.c:                    userViewDelay();
./USB/usb_cmd_parser.c:            userViewDelay();
./UTILS/delays.h:void userViewDelay(void);
./UTILS/delays.c:/*! \fn     userViewDelay(void)
./UTILS/delays.c:void userViewDelay(void)
```

However there is this fancy `guiDisplayInformationOnScreenAndWait()` function which the others could possibly also use as an alternative, where we could add the delay as well. I was also wondering why not all code parts use this function, as it would be possibly the better place to summarize the wait & display stuff.